docs/src/main/sphinx/client/cli.rst

```diff
@@ -102,7 +102,7 @@ allows you to query tables directly without specifying catalog and schema.
 
 .. code-block:: text
 
-  ./trino --server http://trino.example.com:8080 --catalog tpch --schema tiny
```

Comment:
does the previous `--server` still works?

Reply:
Yes, this change is fully compatible. I updated the docs since the new syntax is shorter and easier to type and doesn't require remembering flag names.

Reply:
Ok. I just wanted to make sure that we are not changing the previous behavior.